cypress/integration/home.spec.js

```diff
@@ -8,7 +8,7 @@ describe("Home e2e", function () {
     cy.contains("Your Draft Submissions", { timeout: 10000 }).should("be.visible")
     cy.contains("Your Published Submissions", { timeout: 10000 }).should("be.visible")
 
-    cy.get("ul.MuiList-root").eq(0).children().should("have.length.at.most", 5)
+    // cy.get("ul.MuiList-root").eq(0).children().should("have.length.at.most", 5)
```

Comment:
it's not applicable if I have a cleared database and run the Home test only, because at that time I don't have any folder so there is no list. But if we run all the tests together then we would already have some folders created before that.
I will just keep it. I will uncomment it.